cmd/cri-containerd/options/options.go

```diff
@@ -31,6 +31,8 @@ type CRIContainerdOptions struct {
 	ContainerdSocketPath string
 	// ContainerdConnectionTimeout is the connection timeout for containerd client.
 	ContainerdConnectionTimeout time.Duration
+	// CRIContainerdVersion is the git release version of cri-containerd
+	CRIContainerdVersion bool
```

Comment:
Can we move this up to under `CRIContainerdSocketPath`?

It would be better to group Containerd options together, and CRIContainerd options together. :)